modules/machine-config-daemon-metrics.adoc

```diff
@@ -55,11 +55,7 @@ For further investigation, see the logs by running:
 |Logs errors encountered during pivot. *
 |Pivot errors might prevent OS upgrades from proceeding.
 
-For further investigation, run this command to access the node and see all its logs:
-
-`$ oc debug node/<node> -- chroot /host journalctl -u pivot.service`
```

Comment:
Thanks! Works for me, we don't have pivot as a command anymore.

As a note, this error should be very rare. This should only happen if we were to fail in staging an os update of some sort. In practice, most os upgrade failures happen at a later stage.